tomcat-main/src/main/resources/defaults/base-procedure-nagprainventory.xml

```diff
@@ -0,0 +1,78 @@
+<record id="nagprainventory" in-findedit="yes" type="record,procedure" cms-type="default" generate-services-schema="true" >
+  <services-url>nagprainventories</services-url>
+  <services-tenant-plural>NagpraInventories</services-tenant-plural>
+  <services-tenant-singular>NagpraInventory</services-tenant-singular>
+  <services-list-path>abstract-common-list/list-item</services-list-path>
+  <services-record-path>nagprainventories_common:http://collectionspace.org/services/nagprainventory,nagprainventories_common</services-record-path>
+  <services-record-path id="collectionspace_core">collectionspace_core:http://collectionspace.org/collectionspace_core/,collectionspace_core</services-record-path>
+
+  <include src="domain-procedure-nagprainventory.xml" strip-root="yes" />
+
+  <section id="coreInformation">
+    <include src="core-fields.xml" strip-root="yes" />
+  </section>
+
+  <section id="nagpraInventoryInformation">
+    <field id="inventoryNumber" mini="number,list" />
+    <field id="originationDate" datatype="date" />
+    <repeat id="titles">
+      <field id="title" mini="summary,list" />
+    </repeat>
+    <repeat id="noticeTypes">
+      <field id="noticeType" autocomplete="true" ui-type="enum" />
+    </repeat>
+    <repeat id="consultationNotes">
+      <field id="consultationNote" />
+    </repeat>
+    <repeat id="treatmentNotes">
+      <field id="treatmentNote" />
+    </repeat>
+    <repeat id="partiesInvolvedGroupList/partiesInvolvedGroup">
+      <field id="involvedParty" autocomplete="true" />
+      <field id="involvedOnBehalfOf" autocomplete="true" />
+      <field id="involvedRole" />
```

Comment:
This is using a dynamic term list in the UI, so should have `autocomplete="true" ui-type="enum"`.